build/teamcity/internal/release/build-and-publish-patched-go/diff.patch

```diff
@@ -20,26 +18,40 @@ index 3991a48b10..4a50ae650d 100644
 +		// state.
 +		gp.runningnanos += now - gp.lastsched
```

Comment:
I noticed that a histogram tracking scheduling latencies has been added [1]. Technically, the sum of all latencies is now derivable via `runtime.metrics`. Do we still need to maintain this patch?

[1] https://github.com/golang/go/commit/bedfeed54a7a80123c07f83c325a0bcfe5c43398

Reply:
cc @irfansharif

Reply:
The patch doesn’t measure scheduling latencies, it measures how much running time each goroutine gets on-CPU. Yes, we still need it.

Reply:
Yep, I see that now; thanks.

```
+       if newval == _Grunning {
+               // We're transitioning into the running state, record the timestamp for
+               // subsequent use.
+               gp.lastsched = now
+       }
+
        if oldval == _Grunning {
+               // We're transitioning out of running, record how long we were in the
+               // state.
+               gp.runningnanos += now - gp.lastsched
```